.github/workflows/ask-sdk-controls-unit-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Unit Tests on ask-sdk-controls
+
+on:
+  push:
+    branches:
+      - 2.0.x
+    paths:
+      - 'ask-sdk-controls/**'
+  pull_request:
+    branches:
+      - 2.0.x
+    paths:
+      - 'ask-sdk-controls/**'
+
+defaults:
+  run:
+    working-directory: ask-sdk-controls
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [10, 12]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - run: npm install
+    - run: npm run lint
+    - run: npm run test
```

Comment:
nit. i think it needs to be new line. same for other file.

Reply:
Oh I see, it didn't throw me errors like testing on fork repo though, but yeah I can make that change

Reply:
Updated in latest revision.